### ISSUE TITLE
Thanks for the master weights

### ISSUE THREAD

**Issue by 5525345551** [pangafu/LeelaMasterWeight#2]
hi,

just wanted to send message saying thank you for these leelaMaster weights.
They are great to play against and learn from. 

Do you plan on releasing more when leelaZero gets stronger with more training games.?

What human games do you use, pro, amateur?

**Comment by pangafu** [comment]
Yes , I am still train bigger net with newest leelazero and human sgf.

Human games I use pro and 9d download from network.

**Comment by john45678** [comment]
100% agree with @5525345551 Thanks.

**Comment by wonsiks** [comment]
Thanks too.
